src/enums/EToolsTooltips.ts

```diff
@@ -0,0 +1,8 @@
+export const EToolsTooltips = {
```

Comment:
`EToolsHints` nebo `EToolsTooltipsContent` nebo neco takoveho, `Tooltips` mi evokuje myslenku, ze to vraci element a ne `string`